spark-connector/spark-common/src/main/java/org/apache/gravitino/spark/connector/jdbc/JdbcPropertiesConverter.java

```diff
@@ -63,7 +76,16 @@ public Map<String, String> toSparkCatalogProperties(Map<String, String> properti
 
   @Override
   public Map<String, String> toGravitinoTableProperties(Map<String, String> properties) {
-    return new HashMap<>(properties);
+    if (!supportsTableProperties) {
+      for (String key : properties.keySet()) {
+        if (!"owner".equalsIgnoreCase(key)) {
+          throw new UnsupportedOperationException("Doesn't support table property " + key);
+        }
+      }
+      return new HashMap<>();
```

Comment:
Collections.emptyMap is preferred because it doesn't create an new map every time, but this is not an blocking issue, let's go ahead